projects/packages/publicize/src/class-keyring-service-helper.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * Utilities to interact with a Keyring instance.
+ * Used for Publicize as well as the Site Verification tools.
+ *
+ * @package automattic/jetpack-publicize
+ */
+
+namespace Automattic\Jetpack\Publicize;
+
+use Automattic\Jetpack\Connection\Secrets;
```

Comment:
Should we pull in the connection package in the Social plugin for this to work?

Reply:
My Jetpack provides the connection package to the plugin, so I don't think declaring it a plugin-level dependency would be necessary.